client/app/expenses/components/ItemDisplay.tsx

```diff
@@ -0,0 +1,40 @@
+import { useRouter } from 'next/navigation'
+import { FiEdit } from 'react-icons/fi'
+import { Button } from '@/_components/ui'
+import { DeleteExpense } from './DeleteExpense'
+import { Expense } from '../types'
+
+type ItemDisplayProps = {
+  item: Expense
+  userId: number
+}
+
+const ItemDisplay = ({ item, userId }: ItemDisplayProps) => {
+  const router = useRouter()
+  return (
+    <li
+      className={`flex flex-col gap-2 ${userId !== item.buyerId && 'md:pt-7'}`}
+    >
```

Comment:
Hmm I feel `<div>` is correct instead of `<li>` here

Reply:
Fixed!
https://github.com/HAK2024/fairy-share/pull/113/commits/800ef0f1eb10ff8ccb9dd9b3b8cfaaa2e6593931